packages/components/src/components/banner/banner-styles.ts

```diff
@@ -43,14 +36,22 @@ export const getComponentCss = (isOpen: boolean): string => {
               opacity: 1,
               visibility: 'inherit',
               transform: 'translate3d(0,0,0)',
-              transition: `opacity ${duration} ${easeInQuad},transform ${duration} ${easeInQuad}`,
+              transition: `${getTransition('transform', 'moderate', 'in')}, ${getTransition(
```

Comment:
Now that durations are overridden in e2e tests we shouldn't need any timeouts there, right?